core/web/presenters/log.go

```diff
@@ -0,0 +1,12 @@
+package presenters
+
+type LogResource struct {
+	JAID
+	LogLevel string `json:"logLevel"`
```

Comment:
You can omit the `Log` prefix from these fields as they are already inside a `LogResource`